standards/code-reviews.md

```diff
@@ -26,7 +26,7 @@
     - [ ] Make sure your code changes are under 500 lines of code (break it up into more than 1 PR if it exceeds this)
     - [ ] Your PR should be based from the development branch, and not from other open PRs. If you MUST have stair-stepped PRs for a valid reason, please include justification in the PR and reference to the PRs which this new PR depends on.
 
-2. Submit your pull request and tag potential reviewers. Keep in mind that each reviewer should: 
+2. Submit your pull request and tag potential reviewers. Always include the PR Team as potential reviewers! Keep in mind that each reviewer should: 
     * Be either on your project or a quick study on the stack your project is in
```

Comment:
These three lines should be removed (as discussed with Corey).
He also wanted these items documented:
A quick explanation of what pr-team is, and the fact that it will automatically replace the team with 3 auto-selected developers when the PR is submitted.
Mention that there are stack specific teams (devs-wordpress, devs-react, etc) which will also auto assign reviewers. These ones only assign 2 developers.
Explain the purpose of the pr-team and our approach on the review process:
Cross-team reviews encourage cross-training and knowledge sharing
PR reviewers are just as valuable as a learning opportunity for the reviewer as they are for the reviewee
Automatic assignment balances the workload evenly across the whole team
Everyone gets the opportunity to be both a reviewer and a reviewee
Pending PR reviews show up on the #shift3-code-review Slack channel as a notification and reminder. This has helped us develop a very quick turn-around on reviews.
Our standard code review process only requires a visual inspection of the code. Developers are not required to pull down and run/test the code. That said, project teams can add their own functional review process within the team in addition to the standard cross-team visual review.

Reply:
Hey @lizl714,

Thanks for all that super helpful info, I updated the documentation with the additions. I know there is an ongoing discussion in the issue as to whether the entire pr-team should be tagged on all PR's. So I will make that change as soon as that decision has been finalized. Other than that, if there's something else I can add or a better way to format the info, I would love to get your thoughts.

Thanks again,
Thom